content/docs/next-release/operator.md

```diff
@@ -0,0 +1,645 @@
+simplest-6499bb6cdd-kqx75   1/1       Running   0          2m
+```
+
+Similarly, the logs can be queried either from the pod directly using the pod name obtained from the previous example, or from all pods belonging to our instance:
+
+```bash
+$ kubectl logs -l app.kubernetes.io/instance=simplest
+...
+{"level":"info","ts":1535385688.0951214,"caller":"healthcheck/handler.go:133","msg":"Health Check state change","status":"ready"}
+```
+
+NOTE: On OKD/OpenShift the container name must be specified.
+
+```bash
+$ kubectl logs -l app.kubernetes.io/instance=simplest -c jaeger
+...
+{"level":"info","ts":1535385688.0951214,"caller":"healthcheck/handler.go:133","msg":"Health Check state change","status":"ready"}
+```
+
+For reference, here's how you can create a more complex all-in-one instance:
+
+.all-in-one.yaml
+```yaml
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: my-jaeger
+spec:
+  strategy: allInOne # <1>
+  allInOne:
+    image: jaegertracing/all-in-one:latest # <2>
+    options: # <3>
+      log-level: debug # <4>
+  storage:
+    type: memory # <5>
+    options: # <6>
+      memory: # <7>
+        max-traces: 100000
+  ingress:
+    enabled: false # <8>
+  agent:
+    strategy: DaemonSet # <9>
+  annotations:
+    scheduler.alpha.kubernetes.io/critical-pod: "" # <10>
+```
+
+<1> The default strategy is `allInOne`. The other possible values are `production` and `streaming`.
+
+<2> The image to use, in a regular Docker syntax.
+
+<3> The (non-storage related) options to be passed verbatim to the underlying binary. Refer to the Jaeger documentation and/or to the `--help` option from the related binary for all the available options.
+
+<4> The option is a simple `key: value` map. In this case, we want the option `--log-level=debug` to be passed to the binary.
+
+<5> The storage type to be used. By default it will be `memory`, but can be any other supported storage type (Cassandra, Elasticsearch, Kafka).
+
+<6> All storage related options should be placed here, rather than under the 'allInOne' or other component options.
+
+<7> Some options are namespaced and we can alternatively break them into nested objects. We could have specified `memory.max-traces: 100000`.
+
+<8> By default, an ingress object is created for the query service. It can be disabled by setting its `enabled` option to `false`. If deploying on OpenShift, this will be represented by a Route object.
+
+<9> By default, the operator assumes that agents are deployed as sidecars within the target pods. Specifying the strategy as "DaemonSet" changes that and makes the operator deploy the agent as DaemonSet. Note that your tracer client will probably have to override the "JAEGER_AGENT_HOST" environment variable to use the node's IP.
+
+<10> Define annotations to be applied to all deployments (not services). These can be overridden by annotations defined on the individual components.
+
+
+# Understanding Custom Resource Definitions
```

Comment:
Just wondering whether this section should be before `Creating a Simple Jaeger Instance`?

Reply:
IMO, the doc should begin with a quick-start, and then expand to explain everything else.

Reply:
I'll look at reorganizing things so that the doc starts with a quick-start.